pkg/networkmanager/network-interface.jsx

```diff
@@ -152,9 +152,7 @@ export const NetworkInterfacePage = ({
 
         function modify () {
             return dev.disconnect()
-                    .fail(function (error) {
-                        show_unexpected_error(error);
-                    });
+                    .catch(error => show_unexpected_error(error));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19120-20230721-193803-d973134a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)